src/sqlfluff/core/templaters/base.py

```diff
@@ -0,0 +1,359 @@
+"""Defines the templaters."""
+
+import logging
+from typing import Dict, Iterator, List, Tuple, Optional, NamedTuple
+
+_templater_lookup: Dict[str, "RawTemplater"] = {}
+
+# Instantiate the templater logger
+templater_logger = logging.getLogger("sqlfluff.templater")
+
+
+def templater_selector(s=None, **kwargs):

```

Comment:
This should probably by `s="jinja"` and we can remove `s = s or "jinja"`, that way it's more information from the function signature

Reply:
It's set up that way so that if `None` is explicitly passed at `s` then we still get the jinja templater. The way config is set up, we always get the keyword, but sometimes with `None`. I didn't want to have the default for `jinja` set it several different places, and so this approach means we can set it in just one place and still get the desired effect.